crypto/asn1/a_object.c

```diff
@@ -85,7 +85,7 @@ int a2d_ASN1_OBJECT(unsigned char *out, int olen, const char *buf, int num)
             c = *(p++);
             if ((c == ' ') || (c == '.'))
```

Comment:
The original code isn't converting to/from ASCII, so the new version behaves identically.